app/src/main/java/com/dsvag/currencyexchanger/data/database/CoinDAO.kt

```diff
@@ -0,0 +1,19 @@
+package com.dsvag.currencyexchanger.data.database
+
+import androidx.room.Dao
+import androidx.room.Delete
+import androidx.room.Insert
+import androidx.room.Query
+import com.dsvag.currencyexchanger.data.models.latest.Coin
+
+@Dao
+interface CoinDAO {
```

Comment:
Опять же, принято писать вот так:
```suggestion
interface CoinDao {
```